RELEASE_NOTES.md

```diff
@@ -4,6 +4,7 @@
 - Changes UI to use new dark theme
 - Adds "Open File and Exit" command
 - #92 Adds per-folder storage of sort settings (Thanks @jilleJr)
+- #92 Adds support for `%APPDATA%`-esque input in location field (Thanks @jilleJr)
```

Comment:
This issue is 96. You could describe this as "Adds support for environment variable substitution in path box"

Reply:
Lol oops! And yea that's better phrased.

```suggestion
- #96 Adds support for environment variable substitution in path box (Thanks @jilleJr)
```